src/test/java/hw2/ex1/ElementsOnTheHomePage.java

```diff
@@ -0,0 +1,63 @@
+package hw2.ex1;
+
+import hw2.base.BaseTest;
+
+import hw2.enums.LoginUser;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class ElementsOnTheHomePage extends BaseTest {
+
+    @Test
+    public void thePresenceOfMainElementsOnTheHomePage() {
```

Comment:
Great! this test is exactly what should be. 
Pls make second test in that style